Library/Homebrew/dev-cmd/unbottled.rb

```diff
@@ -20,10 +20,12 @@ def unbottled_args
              description: "Use the specified bottle tag (e.g. `big_sur`) instead of the current OS."
       switch "--dependents",
              description: "Skip getting analytics data and sort by number of dependents instead."
-      switch "--all", "--total",
+      switch "--all",
+             description: "Print the number of unbottled and total formulae. Implied if HOMEBREW_TRUST_ALL is set."
+      switch "--total",
+             hidden:      true,
```

Comment:
Not really convinced this is a better name given this description.

It also does not behave this way when paired with `--dependents`. Flag feels confusing on its own.

Reply:
Same as above re: name.

Reply:
It might be worth keeping `--total` but requiring it to be paired with `--all-whatever`

Reply:
That works 👍🏻